psydac/feec/pushforward.py

```diff
@@ -100,7 +100,10 @@ def __init__(
         self.npts_per_cell = npts_per_cell
         self.cell_indexes = cell_indexes
         self.grid_type=grid_type
-        grid_local=grid_local
+        if grid_local is None:
+            grid_local=grid
+        else:
+            grid_local=grid_local
```

Comment:
These two lines do not seem necessary at all.

The original `grid_local = grid_local` was also useless of course 😉 
```suggestion
```

Reply:
ahah indeed